src/status_im/chat/db.cljs

```diff
@@ -29,15 +32,25 @@
         (assoc :inviter-name inviter-name)
         :always
         (assoc :chat-name (group-chat-name chat))))
-    (let [{contact-name :name :as contact}
+    (let [photo (if (seq identicon)
```

Comment:
why seq ?

Reply:
We could use `blank?` but we just want to check if it's not `""` or `nil`(i.e empty)